releasenotes/notes/v05-encoding-pack-str-error-handle-d11960b72472af7c.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    tracing: This fix resolves an issue where using the v0.5 encoding endpoint did not raise any ``TypeError``
+    if non-text span tags were set.
```

Comment:
Maybe worth rewording to be "v0.5 encoding endpoint errors were output to stderr instead of to a logger..."? The semantics of a user's application isn't affected by this change. I believe the error would have been caught and logged still, it just got printed to stderr directly instead of to a logger.